config.json

```diff
@@ -389,6 +389,45 @@
       "interviewer_handle": "jonathandmiddleton",
       "interviewee_handle": "isaacg",
       "youtube_id": "mAVkVr3S2AM"
+    },
+    {
+      "uuid": "955266e4-f931-4578-bc57-6716b19aa226",
+      "slug": "beating-paralysis-and-printing-a-one-handed-keyboard",
+      "title": "Beating Paralysis and Printing a one-handed Keyboard",
+      "blurb": "What happens when you think that a door which was open, suddenly closes in your face? Bobbi's story of overcoming a severely debilitating injury to continue his journey of being a programming is remarkable!",
+      "published_at": "2022-12-23 17:01",
+      "length_in_minutes": 64,
+      "thumbnail_url": "https://exercism-static.s3.eu-west-1.amazonaws.com/blog/story-bobbi-towers.jpg",
+      "image_url": "https://test.org/image.png",
+      "interviewer_handle": "jonathandmiddleton",
+      "interviewee_handle": "porkostomus",
+      "youtube_id": "IlEDKBPax_g"
+    },
+    {
+      "uuid": "1d4aad9e-b737-4edc-889a-69365b412f59",
+      "slug": "becoming-a-polyglot-developer",
+      "title": "Becoming a Polyglot Developer",
+      "blurb": "What is a 'Polyglot, I hear you say...well, according to the dictionary, it's the ability to know several languages! So how do you become a 'Polyglot'? You talk to Erik Schierboom and get the inside track!",
+      "published_at": "2022-12-30 17:01",
+      "length_in_minutes": 57,
+      "thumbnail_url": "https://exercism-static.s3.eu-west-1.amazonaws.com/blog/story-erik-schierboom.jpg",
+      "image_url": "https://test.org/image.png",
+      "interviewer_handle": "jonathandmiddleton",
+      "interviewee_handle": "erikschierboom",
+      "youtube_id": "EAJqTlojzmU"
+    },
+    {
+      "uuid": "ed226307-ae5b-461f-bfc3-af70e4518745",
+      "slug": "conquering-cancer-to-accessibility-activism",
+      "title": "Conquering Cancer to Accessibility Activism",
+      "blurb": "What happens when you get life-changing news or information? How do you respond? How do you process? Catch up with DJ to hear how he re-framed his perspective on hearing he'd been diagnosed with cancer.",
+      "published_at": "2023-01-06 17:01",
+      "length_in_minutes": 49,
+      "thumbnail_url": "https://exercism-static.s3.eu-west-1.amazonaws.com/blog/story-dj.jpg",
```

Comment:
This image is not on AWS. Did you send it to me?

Reply:
edit: I didn't see your comment, sorry

Reply:
![DJ-Thumbnail](https://user-images.githubusercontent.com/58132850/209068692-eb21e5ea-9199-4a29-837d-d8d3d5bf17dd.png)